waveguide_mode_converter/README.md

```diff
@@ -48,3 +48,145 @@ The script will print out CSV-formatted performance metrics. For example,
 -34.11, -0.19
 -30.67, -0.26
 ```
+
+For comparison, the following are the reflectance and transmittance spectra for the topology-optimized designs using Meep with various minimum lengthscales.
+
+1. Minimum lengthscale constraint: 50 nm.
+
+measured lengthscale: 63 nm.
+
+file: converter_meep_min_linewidth_50nm.csv
+
+**reflectance into mode 1: wavelength (μm), reflectance, reflectance (dB)**
+```
```

Comment:
For compactness, since we may end up with many designs in this repo, we might want to summarize only the worst case values, instead of wavelength-by-wavelength.

Reply:
Done.